scripts/zones/Temenos/mobs/Enhanced_Vulture.lua

```diff
@@ -2,27 +2,24 @@
 -- Area: Temenos W T
 --  Mob: Enhanced Vulture
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+local ID = require("scripts/zones/Temenos/IDs")
 
-function onMobEngaged(mob,target)
-        GetMobByID(16928959):updateEnmity(target);
-        GetMobByID(16928960):updateEnmity(target);
-        GetMobByID(16928961):updateEnmity(target);
-        GetMobByID(16928962):updateEnmity(target);
-        GetMobByID(16928963):updateEnmity(target);
-        GetMobByID(16928964):updateEnmity(target);
-end;
+function onMobEngaged(mob, target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]):updateEnmity(target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]+1):updateEnmity(target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]+2):updateEnmity(target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]+3):updateEnmity(target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]+4):updateEnmity(target)
+    GetMobByID(ID.mob.TEMENOS_W_MOB[7]+5):updateEnmity(target)
```

Comment:
I probably should have asked earlier- and I know this was already here - but can we use standard linking mechanics for these situations?

Reply:
Since they link by sight you'd be able to pull from behind 1 at a time. I think they all attack at once, but may be wrong.